flink-dist/src/test/bin/runBashJavaUtilsCmd.sh

```diff
@@ -34,9 +34,10 @@ bin=`cd "$bin"; pwd`
 
 FLINK_CONF_DIR=${bin}/../../main/resources
 FLINK_TARGET_DIR=${bin}/../../../target
-FLINK_DIST_JAR=`find $FLINK_TARGET_DIR -name 'flink-dist*.jar'`
+FLINK_DIST_JARS=`find ${FLINK_TARGET_DIR} -name 'flink-dist*.jar' -maxdepth 1`
```

Comment:
The `find` command works fine on my Mac, but throws the following warning on my Linux machine.
> find: warning: you have specified the -maxdepth option after a non-option argument -name, but options are not positional (-maxdepth affects tests specified before it as well as those specified after it).  Please specify options before other arguments.

This might not be a big problem since the warning is not in stdout, but sill would be good to fix it.